pkg/kudoctl/cmd/init/crds.go

```diff
@@ -31,6 +35,20 @@ func installCrds(client apiextensionsclient.Interface) error {
 	return nil
 }
 
+func validateCrd(client v1beta1.CustomResourceDefinitionsGetter, crd *apiextv1beta1.CustomResourceDefinition) error {
```

Comment:
crdExists/crdInstalled? it's not really doing any validation of the crd other than that...

Reply:
Well, it does checks that the CRD version is as expected. It doesn't validate the CRD itself, maybe `validateCrdInstallation`?

Reply:
Renamed it to "validateInstallation". No need to mention the Crd in there, as it's an internal function